src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+@use "inc" as *;
+
 body {
+  padding: 0;
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  width: 100%;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: summaxwidth(700, 6);
+  margin: 0 rem(10);
+
+  @each $daycount, $dayname in $weekdays {
+    &--start-day-#{$dayname} {
+      .calendar__item {
+        &:first-child {
+          margin-left: calc(#{$daycount} * ($daywidth + $gap));
+        }
+      }
+    }
+  }
+
+  @for $i from 1 through $calendardays {
+    &--month-length-#{$calendardays} {
+      .calendar__item {
+        &:nth-child(#{$i}) {
+          display: block;
+        }
+      }
+    }
+  }
+
+  &__item {
+    width: rem(100);
+    height: rem(100);
+    position: relative;
+    background-color: #eee;
+    border: rem(1) solid $black;
+    display: none;
```

Comment:
It's better to do it vice versa:
use `display: block` as a default value, but create a rule to hide unnecessarily days (see comment above)